src/ZoneServer/World/Maps/Map.cs

```diff
@@ -137,7 +142,10 @@ private void Load()
 
 			// A few maps don't seem to have any ground data.
 			if (ZoneServer.Instance.Data.GroundDb.TryFind(this.ClassName, out var groundData))
+			{
 				this.Ground.Load(groundData);
+				this.Pathfinder.Load(this.Ground);
```

Comment:
Would it perhaps make more sense to pass the whole map to the pathfinder? Decoupling the two would be nice, but the pathfinder will presumably need to know about monsters, their bounding boxes, and things like platforms down the line, in addition to the ground. Probably easiest to just give it everything it will need. Though we could also add an interface with methods for providing the ground and potential obstacles for maps or other consumers.